src/components/calculation/index.js

```diff
@@ -187,6 +188,50 @@ class Calculation extends Component {
       })
     }
 
+    function getMoleculeView() {
+      // For some reason, the molecule is sometimes null
+      // Do a check here to prevent a javascript error
+      if (molecule === null)
+        return;
+
+      if (has3dCoords(molecule)) {
+        return (
+          <oc-molecule
+            ref={wc(
+              // Events
+              {
+                iOrbitalChanged: onIOrbitalChanged
+              },
+              //Props
+              {
+                cjson: {...cjson, cube},
+                orbitalSelect: true,
+                moleculeRenderer,
+                showIsoSurface,
+                showVolume,
+                showSpectrum,
+                showMenu,
+                isoValue,
+                iOrbital,
+                iMode: mode,
+                play,
+                colors,
+                colorsX,
+                opacities,
+                opacitiesX
+              }
+            )}
+          />
+        )
+      }
+      else {
+        const src = `${window.location.origin}/api/v1/molecules/${molecule._id}/svg`
+        return (
+          <img src={src} class={classes.moleculeContainer}/>
+        )
+      }
+    }
+
```

Comment:
Unlike the molecule version of `getMoleculeView()`, I put this `getMoleculeView()` inside the `render()` function because there are many local variables in the render function that it needs access to, and I thought this would be cleaner than passing all the local variables as arguments.